notebooks/book1/03/sensor_fusion_2d.ipynb

```diff
@@ -0,0 +1,503 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.        # Code below is unused.
</pre><p>Code below was unused, let me know if this can be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Sure</p>